src/Symfony/Component/Finder/Finder.php

```diff
@@ -153,6 +155,31 @@ public function date($dates)
         return $this;
     }
 
+    /**
+     * Adds rules that files must match like $finder->name() but with case-insensitivity set.
+     *
+     * You can use patterns (delimited with / sign), globs or simple strings.
+     *
+     *     $finder->iname('*.php')
```

Comment:
These examples aren't very easy to understand.

Maybe something like:

```php
*     $finder->iname('*.php') // will match both *.php and *.PHP
```

?